presto-main/src/test/java/io/prestosql/sql/query/TestSessionFunctions.java

```diff
@@ -27,7 +27,9 @@
     @Test
     public void testCurrentUser()
     {
-        Session session = testSessionBuilder().setIdentity(new Identity("test_current_user", Optional.empty())).build();
+        Session session = testSessionBuilder()
+                .setIdentity(Identity.forUser("test_current_user").build())
```

Comment:
`Identity.forUser("xxx").build()` is a very common pattern.
Maybe there should be one-shot method to do this?